src/qt/paymentserver.cpp

```diff
@@ -237,7 +237,7 @@ void PaymentServer::handleURIOrFile(const QString& s)
                         Q_EMIT message(tr("URI handling"),
                             tr("Cannot process payment request because BIP70 is not supported.")+
                             tr("Due to widespread security flaws in BIP70 it's strongly recommended that any merchant instructions to switch wallets be ignored.")+
-                            tr("If you are receiving this error you should request the merchant provide a BIP21 compatible URI."),
+                            tr("If you are receiving this error, you should request from the merchant to provide a BIP21 compatible URI."),
```

Comment:
I don't think we should be making people re-translate the BIP70 deprecation message.